<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registers</class>
 <widget class="QDialog" name="Registers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(18, 53, 85);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>231</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 17pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>欢迎光临佳佳鸽饭店！</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>29</number>
      </property>
      <item>
       <widget class="QLabel" name="label_username">
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 155, 3);
font: 17pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_username">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>52</number>
      </property>
      <item>
       <widget class="QLabel" name="label_password">
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 155, 3);
font: 17pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_password_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(230, 155, 3);
font: 17pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(225, 238, 210);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
